server/src/main/java/org/uiautomation/ios/utils/SimulatorSettings.java

```diff
@@ -56,7 +60,7 @@
   private InstrumentsVersion instrumentsVersion;
 
   public SimulatorSettings(HostInfo info, String sdkVersion, boolean is64bit, DeviceType device,
-                           DeviceVariation variation) {
+                           DeviceVariation variation) throws NotActiveException {
```

Comment:
Yes. I see that issue. So you mean we should `throw new SessionNotInitializedException` when the simulator device is not available to use in the line 224.